GMPWrapper/scripts/convertMarlinSteeringToGaudi.py

```diff
@@ -86,7 +87,7 @@ def resolveGroup(lines, proc, execGroup):
           lines.append("algList.append(%s)" % child.get('name').replace(".", "_"))
 
 
-def getExecutingProcessors(lines, tree):
+def getExecutingProcessors(lines, tree, optProcessors=False):
   """ compare the list of processors to execute, order matters """
   execProc = tree.findall('execute/*')
   execGroup = tree.findall('group')
```

Comment:
```suggestion
  execGroup = tree.findall('group')
  optProcessors = False
```
Instead of function argument, set in the function